src/pages/homeCollection/printLabels.js

```diff
@@ -123,15 +123,12 @@ const generateParticipantCsvGetter = (name) => {
 const generateParticipantCsv = async (items) => {
   let csv = ``;
   let participantsForKitUpdate = []
-  csv += `first_name, last_name, address_1, address_2, city, state, zip_code, connect_id, \r\n`
+  const columns = ['first_name', 'last_name', 'address_1', 'address_2', 'city', 'state', 'zip_code', 'connect_id'];
+  csv += `${columns.join(',')}, \r\n`
```

Comment:
Do you need the trailing comma here? Just asking, I haven't tested it.
```suggestion
  csv += `${columns.join(',')} \r\n`
```

Reply:
I suspect it's fine without but I'm including it because it was in the previous version and I don't want to risk it turning out to have been load-bearing somewhere.